projects/ad7768evb/Readme.md

```diff
@@ -0,0 +1,11 @@
+# AD7768EVB HDL Project
+
+Here are some pointers to help you:
+  * [Board Product Page](https://www.analog.com/EVAL-AD7768-1)
+  * [Board Product Page](https://www.analog.com/EVAL-AD7768)
+  * Parts : [DC to 204 kHz, Dynamic Signal Analysis, Precision 24-Bit ADC with Power Scaling](https://www.analog.com/ad7768-1)
+  * Parts : [8-Channel, 24-Bit, Simultaneous Sampling ADC, Power Scaling, 110.8 kHz BW](https://www.analog.com/ad7768)
+  * Project Doc: https://wiki.analog.com/resources/eval/user-guides/ad7768-ebz/software/baremetal
+  * HDL Doc: https://wiki.analog.com/resources/eval/user-guides/ad7768-1
```

Comment:
AD7768 and AD7768-1 are different parts. The links should point to https://wiki.analog.com/resources/eval/user-guides/ad7768-ebz/software/baremetal for HDL. AD7768-1 projects are in the ad77681evb folder